lib/widgets/recent_dm_conversations.dart

```diff
@@ -173,22 +175,46 @@ class RecentDmConversationsItem extends StatelessWidget {
   Widget build(BuildContext context) {
     final store = PerAccountStoreWidget.of(context);
     final designVariables = DesignVariables.of(context);
-
     final InlineSpan title;
     final Widget avatar;
     int? userIdForPresence;
+    TextSpan buildUserTitleSpan(
```

Comment:
This can be a private method on the class; let's not declare it here in the build method. Also, please clean up the indentation.